src/elements/frame-router.ts

```diff
@@ -10,10 +16,14 @@ const ROUTE_ATTR = 'route';
  * the client content.
  */
 class FrameRouterElement extends HTMLElement {
```

Comment:
I would like to add tests for this class, but I haven't been able to inject mocks for dependencies and construct an instance of it in a test successfully (either via `new` or as a custom elment).  In theory I could construct one without intercepting the imports and just monkey-patch the "private" fields, although TypeScript would probably refuse to compile, and it'd be an ugly kludge.